flink-ml-lib/src/test/java/org/apache/flink/ml/classification/LogisticRegressionTest.java

```diff
@@ -0,0 +1,120 @@
+    static StreamTableEnvironment tableEnv = StreamTableEnvironment.create(env);
+
+    @Test
+    public void test() throws Exception {
+        List<Row> rows =
+                Arrays.asList(
+                        Row.of("1 2 3 4", 1.),
+                        Row.of("1 2 3 4", 1.),
+                        Row.of("1 2 3 4", 1.),
+                        Row.of("1 2 3 4", 1.),
+                        Row.of("1 2 3 4", 1.),
+                        Row.of("3 2 3 4", -1.),
+                        Row.of("1 2 3 4", 1.),
+                        Row.of("3 2 3 4", -1.));
+
+        Table input =
+                tableEnv.fromDataStream(
+                        env.fromCollection(
+                                rows,
+                                new RowTypeInfo(
+                                        new TypeInformation[] {Types.STRING, Types.DOUBLE},
+                                        new String[] {"vec", "label"})));
+
+        LogisticRegression logisticRegression =
+                new LogisticRegression()
+                        .setVectorCol("vec")
+                        .setLabelCol("label")
+                        .setMaxIter(10)
+                        .setEpsilon(0.1)
+                        .setLearningRate(1.)
+                        .setBatchSize(100)
+                        .setL1(0.1)
+                        .setL2(0.1)
+                        .setWithIntercept(true)
+                        .setOptimMethod(OptimMethod.SGD)
+                        .setStandardization(true);
+        LogisticRegressionModel logisticRegressionModel = logisticRegression.fit(input);
+
+        Table pred =
+                logisticRegressionModel
+                        .setPredictionCol("pred")
+                        .setPredictionDetailCol("predDetail")
+                        .transform(input)[0];
+
+        // test training
+        tableEnv.toDataStream(logisticRegressionModel.getModelData()[0])
+                .map(row -> (double[]) row.getField(0))
+                .addSink(
+                        new SinkFunction<double[]>() {
+                            @Override
+                            public void invoke(double[] value, Context context) throws Exception {
+                                System.out.println(
+                                        "The output model is: " + Arrays.toString(value));
+                                System.out.println("[Output] model: " + Arrays.toString(value));
+                                Assert.assertArrayEquals(
+                                        new double[] {-2.55, 0.41, 0.63, 0.84}, value, 0.01);
+                            }
+                        });
+        // test prediction
+        String vecCol = logisticRegressionModel.getVectorCol();
+        String predCol = logisticRegressionModel.getPredictionCol();
+        String predDetailCol = logisticRegressionModel.getPredictionDetailCol();
+        tableEnv.toDataStream(pred)
```

Comment:
Any chance we can add tests that verify the output numerical correctness? Maybe follow Spark's LogisticRegressionSuite for examples.

We will need to remove `System.out` from tests before merging this PR. Otherwise the console output of `mvn test` would be too verbose to read.

Reply:
Thanks for the review. I have updated the PR and (a) added numerical correctness (b) removed system out.